sentry_sdk/integrations/quart.py

```diff
@@ -71,18 +77,62 @@ def setup_once():
         got_request_exception.connect(_capture_exception)
         got_websocket_exception.connect(_capture_exception)
 
-        old_app = Quart.__call__
+        patch_asgi_app()
+        patch_scaffold_route()
+
+
+def patch_asgi_app():
```

Comment:
can you split quart out into a separate PR? We can get the quart maintainer to take a look as well then so he's aware of the changes.

Reply:
https://github.com/getsentry/sentry-python/pull/1830